src/test/java/GmailUtilityTest.java

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.sps.utility.GmailUtility;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Test Gmail Utility functions TODO: Mock the Gmail Service features (Issue #5) */
+@RunWith(JUnit4.class)
+public final class GmailUtilityTest {
+
+  // Units of time for email query string
+  private static final String DAYS_UNIT = "d";
+  private static final String HOURS_UNIT = "h";
+  private static final String DEFAULT_UNIT = "";
+  private static final String INVALID_UNIT = "z";
+  private static final int ONE_UNIT_OF_TIME = 1;
+  private static final int DEFAULT_UNIT_OF_TIME = 0;
+  private static final int INVALID_UNIT_OF_TIME = -100;
+
+  // Values to toggle filtering unread vs all emails in email query string
+  private static final Boolean RETURN_UNREAD_ONLY = true;
+  private static final Boolean DEFAULT_UNREAD_FILTER = false;
+
+  // Sample emails for email query string
+  private static final String SAMPLE_EMAIL = "example@example.com";
+  private static final String DEFAULT_EMAIL = "";
+
+  // Expected email query strings
+  private static final String EMPTY_QUERY = "";
+  private static final String ONE_DAY_QUERY =
+      String.format("newer_than:%d%s ", ONE_UNIT_OF_TIME, DAYS_UNIT);
+  private static final String ONE_HOUR_QUERY =
+      String.format("newer_than:%d%s ", ONE_UNIT_OF_TIME, HOURS_UNIT);
+  private static final String UNREAD_EMAILS_QUERY = "is:unread ";
+  private static final String FROM_EMAIL_QUERY = String.format("from:%s ", SAMPLE_EMAIL);
+
+  @Test
+  public void getQueryStringDays() {
+    // Inputting a query for 1 day should result in a oneDayQuery
```

Comment:
FFT: You already defined the constant `ONE_DAY_QUERY` to improve readability. Is the comment necessary?
(here and below)

Reply:
Removing, but is it okay to have a test function without a comment? I personally think it's pretty clear what the test is doing, but it might not be as obvious for new users of the code. Is it okay to assume they should be able to figure this out?

Reply:
Yeah, it's pretty common to have tests without comments. The "comment" is usually its name. If you want to take a look at guava project tests: https://github.com/google/guava/tree/master/guava-tests/test/com/google/common. (Naturally, sometimes it is necessary to add comments to clarify something, but it's not mandatory to always have comments.)

Reply:
Cool :)